walkthroughSteps/code-along-mentor-notes.md

```diff
@@ -223,7 +223,61 @@ Let's first write a file that gets our information from the database.
           response.end(dynamicData);
         });
     ```
-    
+
     - `getData` is asynchronous, so `response.end` should be inside it, so it doesn't run before the data comes back from the database request (same as an API request).
 
 7. Navigate to `http://localhost:3000/dynamic` to check it's worked.
+
+
+- Now, let's write another function that inserts data into the database.
+
+
+1. Inside `src/dynamic.js`, write an asynchronous function `insertData` that takes a superhero information and a callback, and returns the callback function:
+```js
+const insertData = (superHeroInfo, cb) => {
```

Comment:
There maybe needs to be some more explanation as to when this new function would be actually be called (i.e. in the handler for some post request, after the user fills out some form). And where the `superHeroInfo` comes from.

I guess it would expand the workshop too much to include actual functionality for making and handling a POST request, so I think there at least needs to be an explanation of why we are writing this function.